docker-compose-dev-linux.yml.erb

```diff
@@ -0,0 +1,11 @@
+version: '2.3'
```

Comment:
Why does this config file have an `erb` extension while `docker-compose-dev-macos-delegated.yml` doesn't?

Reply:
Good point - no templating needed for this file.